unit-tests/CMakeLists.txt

```diff
@@ -30,6 +30,7 @@ endif()
 set (unit_tests_sources
     unit-tests-live.cpp
     unit-tests-hdr.cpp
+    unit-tests-alt-ir.cpp
```

Comment:
It shouldn't be part of live tests.
Please use the unit-test framework.

Reply:
Done